### PR TITLE
doc: Blog post on cookiecutter-tesseract announcement

### PR DESCRIPTION
**Pull request by samalipio**
#### Description of changes

Adds a blog post highlighting the recent guide on composing multi-Tesseract pipelines and repackaging the cookiecutter-tesseract announcement from the forum.

#### Testing done

👀

### CODE REVIEW

**Review by dionhaefner — APPROVED** [review]

**Comment by samalipio** [comment]
@PasteurBot I have read the CLA Document and I hereby sign the CLA

File: docs/blog/2026-08-13-cookiecutter-tesseract.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+# From inside the generated project:
+make new mytess RECIPE=jax   # scaffold a component
+make build                   # build all components
+make test                    # test components + app
+make run                     # run the pipeline end-to-end
+```
+
+## Build a multi-Tesseract pipeline, compete for prizes
+
+We're running a virtual hackathon now through August 31, 2026, with $20,000 in cash prizes and research collaboration opportunities. Your challenge? Compose your own differentiable scientific workflow from multiple Tesseracts and use end-to-end gradients to solve a real design, inference, or training problem. [Register today](https://pasteurlabs.ai/tesseract-hackathon-2026/) and try out `cookiecutter-tesseract` to get a head start.
```
Comment (jpbrodrick89) [inline docs/blog/2026-08-13-cookiecutter-tesseract.md]:
```suggestion
We're running a virtual hackathon now through August 31, 2026, with $20,000 in cash prizes and research collaboration opportunities. Your challenge? Compose your own differentiable scientific workflow from multiple Tesseracts and use end-to-end gradients to solve a real design, inference, or training problem. [Register today](https://pasteurlabs.ai/tesseract-hackathon-2026/) and try out `cookiecutter-tesseract` to get a head start.

```

File: docs/blog/2026-08-13-cookiecutter-tesseract.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+---
+
+# An easier way to build your first multi-Tesseract pipeline
+
+A single Tesseract packages one computation unit into a standardized component. If you're building a training loop, optimization routine, or a full application, you'll probably need to chain multiple Tesseracts into a larger workflow or integrate them with other code. To make that easier, we've published a [new guide](https://docs.pasteurlabs.ai/projects/tesseract-core/latest/content/how-to/pipelines/) and the [`cookiecutter-tesseract`](https://github.com/pasteurlabs/cookiecutter-tesseract) template.
+
+## You've built your Tesseracts, now what?
+
+Real-world projects usually involve multiple components, for example a mesher feeding a solver, an encoder feeding a model, a simulation feeding a post-processor. Once you've packaged your code as Tesseracts, the next step is to determine how you'll call and chain your components together. Our new guide on [composing Tesseracts into pipelines](https://docs.pasteurlabs.ai/projects/tesseract-core/latest/content/how-to/pipelines/) walks you through three important considerations as you build:
```
Comment (jpbrodrick89) [inline docs/blog/2026-08-13-cookiecutter-tesseract.md]:
```suggestion
Real-world projects usually involve multiple components, for example a mesher feeding a solver, an encoder feeding a model, a simulation feeding a post-processor. Once you've packaged your code as Tesseracts, the next step is to determine how you'll call and chain your components together. Our new guide on [composing Tesseracts into pipelines](https://docs.pasteurlabs.ai/projects/tesseract-core/latest/content/how-to/pipelines/) walks you through three important considerations as you build:
```

File: docs/blog/2026-08-13-cookiecutter-tesseract.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+2. **Chaining Tesseracts:** How should data flow between Tesseracts, and how can you debug when the chain misbehaves?
+3. **Building a multi-Tesseract pipeline:** How can you structure a complex project to keep things organized and consistent, share code between components, and build/test your components as a set?
+
+## Introducing cookiecutter-tesseract
+
+Tesseract Core is deliberately unopinionated about what you build on top, but once you’re chaining several components into an application, you end up hand-rolling the same things every time: a directory layout, a place for shared utilities, per-component test cases, CI that builds everything, and a runner to tie the pipeline together.
+
+The [`cookiecutter-tesseract`](https://github.com/pasteurlabs/cookiecutter-tesseract) template gives you that structure as a batteries-included starting point, so you don't have to assemble it by hand. If you’re building an app out of a single Tesseract, `tesseract init` is still the right tool; if you’re building one out of many, you may want to consider this template in addition to the above guide.
+
+### What's inside
```
Comment (jpbrodrick89) [inline docs/blog/2026-08-13-cookiecutter-tesseract.md:29]:
```suggestion
### What's inside

```
